pkg/pillar/devicenetwork/arp.go

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2017-2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package devicenetwork
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"syscall"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+type arpPort struct {
```

Comment:
Why two structs with identical information? Wouldn't a single arpInfo struct make sense?

Reply:
Sure, we can have a single structure type. I started that way and then I myself got confused at times while writing code to create ARP entries from uplink port info. So, I thought creating two different structures (with identical content) makes the code easier to read.

Reply:
Having comments to explain how they are used would make things more clear. Otherwise it looks like accidental duplication.

Reply:
will add comment

Reply:
Done